cfme/tests/storage/test_object_store_object.py

```diff
@@ -42,55 +42,7 @@ def storage_object(appliance, provider):
 
 
 @test_requirements.tag
-def test_object_add_remove_tag(storage_object):
-    """
-    Polarion:
-        assignee: anikifor
-        initialEstimate: 1/4h
-        casecomponent: Tagging
-    """
-    # add tag with category Department and tag communication
-    added_tag = storage_object.add_tag()
-    tag_available = storage_object.get_tags()
-    assert tag_available[0].display_name == added_tag.display_name
-    assert tag_available[0].category.display_name == added_tag.category.display_name
-
-    # remove assigned tag
-    storage_object.remove_tag(added_tag)
-    tag_available = storage_object.get_tags()
-    assert not tag_available
-
-
-@pytest.mark.manual
-@test_requirements.storage
-def test_storage_object_store_object_remove():
-    """
-    Requires:
-        OpenstackProvider
-
-    Polarion:
-        assignee: mmojzis
-        casecomponent: Cloud
-        caseimportance: medium
-        initialEstimate: 1/8h
-        startsin: 5.8
-        testSteps:
-            1. Navigate to Object Store Object [Storage > Object Storage > Object
-            Store Objects]
-            2. Select objects for removal
-            3. Remove [Configuration > Remove Object Storage Objects from
-            Inventory]
-        expectedResults:
-            1.
-            2.
-            3. Verify object removed or not
-    """
-    pass
-
-
-@pytest.mark.manual
-@test_requirements.storage
-def test_storage_object_store_object_edit_tag_openstack():
+def test_storage_object_store_object_add_remove_tag_openstack(storage_object):
```

Comment:
Since this test is functionally the same as `test_storage_object_store_container_add_remove_tag_openstack`, just the object you're tagging is different. Would you want to parametrize the test against the object type?

Reply:
Well I am not owner of Tagging FA. I just deduplicated that test as it was multiple times in integration tests.

Reply:
I agree with Matous, let's have it separately as he's not supposed to refactor those tests :)